README.md

```diff
@@ -17,7 +17,7 @@
 
 # About Layer5
 
-[Layer5](https://layer5.io)'s cloud native application and infrastructure management software enables organizations to expect more from their infrastructure. We embrace _developer_-defined infrastructure. We empower developers to change how they write applications, support _operators_ in rethinking how they run modern infrastructure, and enable _product owners_ to regain full-control over their product portfolio.
+[Layer5](https://layer5.io)'s cloud native application and infrastructure management software enables organizations to expect more from their infrastructure. We embrace _developer_-defined infrastructure. We empower developers to change how they write applications, support _operators_ in rethinking how they run modern infrastructure and enable _product owners_ to regain full-control over their product portfolio.
```

Comment:
We intentionally apply the series comma (otherwise known as the Oxford comma or Harvard comma) in our attempt to adhere to [APA style](https://en.wikipedia.org/wiki/APA_style).

This change can be reverted.

Reply:
Look over your other suggestions to insert commas, this first suggestion to remove the comma seems ill-placed and leads me to ponder the sincerity of this PR as something more than profile building in nature.

Reply:
Sure @leecalcote. I'll revert this change. I just researched about the oxford comma and found out that it's correct both ways(with or without the comma). You're right!

Reply:
We are using that comma here, so you’ll want to revert this change.